### ISSUE TITLE
Add Operator data to Operation form for internal users

### ISSUE THREAD

**Issue by andrea-williams** [bcgov/cas-registration#551]
#### Description:

When an internal user is reviewing the data for an individual Operation with a status of "Pending", the internal user will need to see info about the Operator the Operation is connected to. 

See wireframe: 
https://www.figma.com/file/LsDmLDCdnJqI8UahyqvOD8/OBPS-App-Designs?type=design&node-id=159%3A5584&mode=design&t=1iuTzEZytKIN1p0m-1

#### Acceptance Criteria:

Given I am an authenticated `cas_admin` or `cas_analyst` user
When I click the "View Details" button next to an Operation in the Operations table
Then the page I am redirected to should display the Operator info (as seen in wireframe)

Given I am an authenticated `industry_user`
When I click the "View Details" button next to an Operation in my table of Operations
Then the page I am redirected to should NOT display my Operator info

#### Development Checklist:

- [ ] Checklist item
- [ ] Checklist item
- [ ] Checklist item
- [ ] Meets the DOD

**Definition of Ready** (Note: If any of these points are not applicable, mark N/A)

- [ ] User story is included
- [ ] User role and type are identified
- [ ] Acceptance criteria are included
- [ ] Wireframes are included (if required)
- [ ] Design / Solution is accepted by Product Owner
- [ ] Dependencies are identified (technical, business, regulatory/policy)
- [ ] Story has been estimated (under 13 pts)

·**Definition of Done** (Note: If any of these points are not applicable, mark N/A)

- [ ] Acceptance criteria are tested by the CI pipeline
- [ ] UI meets accessibility requirements
- [ ] Configuration changes are documented, documentation and designs are updated
- [ ] Passes code peer-review
- [ ] Passes QA of Acceptance Criteria with verification in Dev and Test
- [ ] Ticket is ready to be merged to main branch
- [ ] Can be demoed in Sprint Review
- [ ] Bugs or future work cards are identified and created
- [ ] Reviewed and approved by Product Owner

#### Dependencies

- Blocked by #612 
- Blocking

**Comment by andrea-williams** [comment]
Reopening - Operator info IS in the wireframe for Operation form from the internal user's view.

**Comment by andrea-williams** [comment]
It's probably best to wait until #612 is completed before picking this up - there will be a LOT of rebasing conflicts otherwise.

**Comment by Dianadec** [comment]
Hey team! Please add your estimates: https://planningpokeronline.com/zz1T5DgYXidDPQtTKck2/ @andrea-williams  @dleard  @marcellmueller  @shon-button @Sepehr-Sobhani  @BCerki

**Comment by BCerki** [comment]
Question before I estimate--when the operator isn't approved yet, the form approves operator and the three steps of the operation form all at once. If we get another operation later from the same now-approved operator, are we changing the form to not show the already-approved operator anymore?

**Comment by Dianadec** [comment]
Perhaps someone can help Brianna's question? @andrea-williams @hannavovk @nanyangpro 

> Question before I estimate--when the operator isn't approved yet, the form approves operator and the three steps of the operation form all at once. If we get another operation later from the same now-approved operator, are we changing the form to not show the already-approved operator anymore?

**Comment by BCerki** [comment]
> Given I am an authenticated `industry_user`
> When I click the "View Details" button next to an Operation in my table of Operations
> Then the page I am redirected to should display my Operator info

bump!

**Comment by andrea-williams** [comment]
oh shiver me timbers, I missed all the email notifications about this! Sorry!

My understanding is that the Operator info will still be displayed. Please correct me if I'm wrong @nanyangpro / @hannavovk
